app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+from __future__ import annotations
```

Comment:
why do you need this import?

Reply:
Idk, maybe it was needed before my final code and I forgot about this.